ls8/ls8.py

```diff
@@ -3,9 +3,9 @@
 """Main."""
 
 import sys
-from cpu import *
+from cpu import CPU
 
 cpu = CPU()
 
-cpu.load()
+cpu.load(sys.argv[1])
```

Comment:
You're passing `sys.argv[1]` as a parameter to lead, but it is never being used. I would either remove `sys.argv[1]` from here and remove the parameter `filename` in `cpu.load` or replace `sys.argv[1]` with `filename` in `cpu.load`